app/dashboard/views.py

```diff
@@ -110,16 +110,26 @@
 
 
 @protected_resource()
-@login_required()
 def oauth_connect(request, *args, **kwargs):
     active_user_profile = Profile.objects.filter(user_id=request.user.id).select_related()[0]
-
+    from marketing.utils import should_suppress_notification_email
     user_profile = {
         "login": active_user_profile.handle,
         "email": active_user_profile.user.email,
         "name": active_user_profile.user.get_full_name(),
         "handle": active_user_profile.handle,
-        "id": active_user_profile.user.id,
+        "id": f'{active_user_profile.user.id}',
+        "auth_data": f'{active_user_profile.user.id}',
+        "auth_service": "gitcoin",
+        "notify_props": {
+            "email": False if should_suppress_notification_email(active_user_profile.user.email, 'chat') else True,
```

Comment:
@owocki is there any place else we should check to make sure we don't get any GDPR complaints from these emails?

Reply:
for sanity we have changed this to be false always, users have to opt in in the chat client to receive emails.

We'll revist making this dynamic once we have more testing completed.